openvino_xai/explainer/explanation.py

```diff
@@ -149,6 +150,65 @@ def save(self, dir_path: Path | str, name: str | None = None) -> None:
             image_name = f"{save_name}_target_{target_name}.jpg" if save_name else f"target_{target_name}.jpg"
             cv2.imwrite(os.path.join(dir_path, image_name), img=map_to_save)
 
+    def plot(self, targets: np.ndarray | List[int | str], backend="matplotlib") -> None:
+        """
+        Plots saliency maps using matplotlib.
+
+        This function takes a list or array of target class indices or names, retrieves the corresponding saliency maps,
+        and plots them using matplotlib. If a provided class index is not available among the saliency maps, a message
+        is printed.
+
+        Args:
+            targets (np.ndarray | List[int | str]): A list or array of target class indices or names to plot.
+            backend (str): The plotting backend to use. Can be either 'matplotlib' (recommended for Jupyter)
+            or 'cv' (recommended for python). Default is 'matplotlib'.
+        """
```

Comment:
AFAIK, the `targets` is given in the initializer when users call `Explainer.explain()`. Do we need to specify it again? Can we even override?
For example, if we call `explanation = Explainer.explain(..., targets=3)`, we cannot call `explanation.plot(targets=10)`.

Reply:
What do you think of removing the `targets` parameter and use `Explanation.targets` property which is already defined?

Reply:
Good point. To check all available saliency maps you can run `explanation.plot(targets=-1)` but I like your suggestion better.

I thought of target parameters in case if we run `explanation = Explainer.explain(..., targets=[0,1,2,3,4,5,6])` and want to visualize only subset:
`explanation.plot(targets=[1,6])`

So I can mark targets as an optional parameter to cover that usecase